### ISSUE TITLE
Hosts / Network View - Chart contains no data

### ISSUE THREAD

**Issue by pnash01** [ntop/ntopng#692]
Hi all,
  I'm running 2.5.160815 - Pro Small Business Edition.  Ntop has been running for the past 28 minutes - when I click on 'chart' icon under the Hosts/Network view, the graphs contain no data, even if I select 5m or 10m as a timeframe.   The Network page does properly display statistics however - over 200gb of traffic observed, etc. 

Thanks,
 -Paul

**Comment by lucaderi** [comment]
@pnash01 Can you please post some screenshots? Do you see anything wrong under Home -> Runtime status ?

**Comment by pnash01** [comment]
Oddly enough, data started to appear in the chart.  I have chart data from 12:13pm to 1:22pm.   Here is the runtime status:

```
15/Aug/2016 13:07:38 [Lua.cpp:5041] WARNING: Script failure [/usr/share/ntopng/scripts/callbacks/minute.lua][/usr/share/ntopng/scripts/callbacks/minute.lua:114: mmaping file '/logstash/ntop-data/9/rrd/ShoutCast.rrd': Invalid argument]
15/Aug/2016 13:00:01 [Lua.cpp:5041] WARNING: Script failure [/usr/share/ntopng/scripts/callbacks/hourly.lua][...ng/pro/scripts/lua/modules/top_scripts/top_aggregate.lua:110: attempt to index a nil value]
15/Aug/2016 11:00:00 [Lua.cpp:5041] WARNING: Script failure [/usr/share/ntopng/scripts/callbacks/hourly.lua][...ng/pro/scripts/lua/modules/top_scripts/top_aggregate.lua:110: attempt to index a nil value]
15/Aug/2016 09:53:03 [NetworkInterface.cpp:1119] WARNING: Use: sudo ethtool -K myri:A2P0 gro off gso off tso off
15/Aug/2016 09:53:03 [NetworkInterface.cpp:1117] WARNING: If you have TSO/GRO enabled, please disable it
15/Aug/2016 09:53:03 [NetworkInterface.cpp:1116] Invalid packet received [len: 1516][MTU: 1514].
15/Aug/2016 09:53:02 [MySQLDB.cpp:590] Succesfully connected to MySQL [blahblahblah.edu ntopng] for interface myri:A2P0
15/Aug/2016 09:53:02 [MySQLDB.cpp:550] Attempting to connect to MySQL for interface myri:A2P0...
```

**Comment by lucaderi** [comment]
@pnash01 @simonemainardi Phil, Simone sill look at the warnings when back from vacation.

**Comment by pnash01** [comment]
I noticed that the charts under the 'Report' tab are also (mostly) empty.   I'm guessing that the minute.lua & hourly.lua contribue to that data.

**Comment by simonemainardi** [comment]
@pnash01 is this the same issue you are experiencing? https://github.com/ntop/ntopng/issues/678

**Comment by simonemainardi** [comment]
@pnash01 do you have any update on this?

**Comment by simonemainardi** [comment]
closing for inactivity. Potentially solved in #678
